MTA-5305 Release Notes for 7.3.2 (#168)
migtools/mta-documentation: docs/topics/ref_fixed-issues-7-3-2.adoc

:_newdoc-version: 2.18.3
:_template-generated: 2025-05-12

:_mod-docs-content-type: REFERENCE

[id="fixed-issues-7-3-2_{context}"]
= Fixed issues

> This section provides the following bug fixes and resolved issues that have a significant impact on the {ProductFullName} 7.3.2.

Reviewer: ```suggestion
This section provides the following issues that have been fixed in the {ProductFullName} 7.3.2.
```

> This section provides the following bug fixes and resolved issues and other problems that have a significant impact on the {ProductFullName} 7.3.2.

Reviewer 2: ```suggestion
This section provides the list of bug fixes, resolved issues, and other problems that have a significant impact on {ProductFullName} 7.3.2.
```

Author: Thanks, I have accepted "up to" instead of "till".

.{ProductShortName} now generates consistent results after analyzing large .EAR binary files 

> Previously, when analyzing binary applications, MTA relied on the Maven search index to determine whether dependencies are internal to the application or whether the application was installed as an open-source dependency. Depending on the availability of the Maven search index, it was assumed that an open-source dependency was internal to the application and includes the dependency in the analysis results. In this case, analysis returned more incidents and greater effort to address issues found in dependencies, even if the user did not expect to see those results. This issue has been resolved in {ProductShortName} 7.3.2. link:https://issues.redhat.com/browse/MTA-5588[(MTA-5588)]

Reviewer 2: ```suggestion
Previously, when analyzing binary applications, {ProductShortName} relied on the Maven search index to determine whether dependencies are internal to the application or whether the application was installed as an open-source dependency. 

Depending on the availability of the Maven search index, it was assumed that an open-source dependency was internal to the application and the dependency was included in the analysis results. In this case, analysis returned more incidents and involved greater effort to address issues found in dependencies, even if the user did not expect to see those results. This issue has been resolved in {ProductShortName} 7.3.2. link:https://issues.redhat.com/browse/MTA-5588[(MTA-5588)]
```

.{ProductShortName} does not generate false positives when analyzing Java applications

> Previously, {ProductShortName} generated a false positive issue after analyzing a Java application dependency. This issue has been resolved in {ProductShortName} 7.3.2. link:https://issues.redhat.com/browse/MTA-4025[(MTA-4025)]

Reviewer: I don't think we need to clearly specify the versin here, as the RN will be placed under the 7.3.2 section + RH SSH RN guidelines.

```suggestion
Before this update, {ProductShortName} generated a false positive issue after analyzing a Java application dependency. With this update, the issue has been fixed. link:https://issues.redhat.com/browse/MTA-4025[(MTA-4025)]
```

Author: Let's please be consistent with such usages in our release notes.
